geowebcache/kml/src/main/java/org/geowebcache/service/kml/KMLService.java

```diff
@@ -467,7 +479,7 @@ private String getMimeTypeOverride(ConveyorKMLTile tile) {
      * @return
      * @throws ServiceException
      */
-    private static String createOverlay(ConveyorKMLTile tile, boolean isPackaged)
+    private String createOverlay(ConveyorKMLTile tile, boolean isPackaged)
```

Comment:
As an alternative to making this non-static, couldn't you just pass in the security dispatcher? (either way seems fine to me)

Reply:
I considered it but this felt more right.